VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/config/E2EConfig.java

```diff
@@ -0,0 +1,6 @@
+package org.edx.mobile.test.feature.config;
+
+public class E2EConfig {
+    public static String EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
>Can you use what's here https://openedx.atlassian.net/wiki/display/MA/App+Testing+Accounts to setup test accounts?

Isn't iOS using the same ?
https://github.com/edx/edx-app-ios/blob/17fa26188ed25914c453f5276a2d8cf23a7c7b4a/Test/EndToEnd/Code/EndToEndConfig.swift#L16

>And the password can also be a uuid. Since we run those against prod it's nice to use something with some randomness in the password

noted.